tests/cmd/stability/main.go

```diff
@@ -43,14 +43,27 @@ func main() {
 	tidbVersion := conf.GetTiDBVersionOrDie()
 	upgardeTiDBVersions := conf.GetUpgradeTidbVersionsOrDie()
 
+	// start a http server in goruntine
+	go func() {
```

Comment:
StartValidatingAdmissionWebhookServerOrDie()

Reply:
config is just associate with wehook and k8s action , so register is better.